backend/data/src/main/kotlin/io/tolgee/security/ratelimit/RateLimitService.kt

```diff
@@ -139,6 +139,23 @@ class RateLimitService(
     )
   }
 
+  fun getIEmailVerificationIpRateLimitPolicy(
+    request: HttpServletRequest,
+    email: String?,
+  ): RateLimitPolicy? {
+    if (!rateLimitProperties.emailVerificationRequestLimitEnabled || email.isNullOrEmpty()) return null
+
+    val ip = request.remoteAddr
+    val key = "global.ip.$ip.email.$email::auth"
```

Comment:
So If I resend it from other IP, I can bypass the rate limit? Why so?

Reply:
I did this because I thought that if a user entered the wrong email and sent the verification N times (so he has a rate limit), he could change the email and send the verification more times, but you correctly noticed the problem with different IPs